### PR TITLE
[webapi][xwalk-3731] Add 8 new tests for WebSpeech

### PR DESCRIPTION
**Pull request by haoxli**
- Add 8 missed check points
- Remane test file
- Sort tests entry
- Fail reason: Network issue, pass on chrome 42.

Impacted tests(approved): new 8, update 0, delete 0
Unit test platform: [Nexus 5][Crosswalk 13.41.309.0]
Unit test result summary: pass 7, fail 1, block 0

### CODE REVIEW

File: webapi/webapi-webspeech-w3c-tests/webspeech/SpeechRecognitionEvent_basics.html [hunk 1]
```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<!--
+Copyright (c) 2014 Intel Corporation.
```
Comment (zqzhang) [inline webapi/webapi-webspeech-w3c-tests/webspeech/SpeechRecognitionEvent_basics.html]: s/2014/2015/g

File: webapi/webapi-webspeech-w3c-tests/webspeech/SpeechRecognitionEvent_basics.html [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+ARE DISCLAIMED. IN NO EVENT SHALL INTEL CORPORATION BE LIABLE FOR ANY DIRECT,
+INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
+OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE,
+EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+Authors:
+        Wu. Xiaojun <xiaojunx.a.wu@intel.com>
```
Comment (zqzhang) [inline webapi/webapi-webspeech-w3c-tests/webspeech/SpeechRecognitionEvent_basics.html]: Is this true?

File: webapi/webapi-webspeech-w3c-tests/webspeech/SpeechRecognitionEvent_results-manual.html [hunk 3]
```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<!--
+Copyright (c) 2013 Intel Corporation.
```
Comment (zqzhang) [inline webapi/webapi-webspeech-w3c-tests/webspeech/SpeechRecognitionEvent_results-manual.html]: s/2013/2015/g

File: webapi/webapi-webspeech-w3c-tests/webspeech/SpeechRecognitionEvent_results-manual.html [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+ARE DISCLAIMED. IN NO EVENT SHALL INTEL CORPORATION BE LIABLE FOR ANY DIRECT,
+INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
+OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE,
+EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+Authors:
+        Shentu. Jiazhen <jiazhenx.shentu@intel.com>
```
Comment (zqzhang) [inline webapi/webapi-webspeech-w3c-tests/webspeech/SpeechRecognitionEvent_results-manual.html]: Where this test come from?

Comment (haoxli) [inline webapi/webapi-webspeech-w3c-tests/webspeech/SpeechRecognitionEvent_results-manual.html]: These tests exist in test suite, but not add in tests.xml, so remove them in https://github.com/crosswalk-project/crosswalk-test-suite/pull/1941, I check some check ponits still useful, so update them follow new spec.

**Comment by haoxli** [comment]
These tests exist in test suite, but not add in tests.xml, so remove them in #1941, I check some check ponits still useful, so update them follow new spec.

**Pull request merged**